### ISSUE TITLE
Issue deploying guestbook app

### ISSUE THREAD

**Issue by michelleN** [kubernetes/helm#186]
I'm getting this timeout error when trying to deploy the guestbook app (following the readme). I'm running on GCE.

``` 
2016/01/26 10:23:34 cannot deploy configuration named guestbook.yaml: Post http://localhost:8001/api/v1/proxy/namespaces/dm/services/manager-service:manager/deployments: net/http:  request canceled (Client.Timeout exceeded while awaiting headers)
```

**Comment by vaikas-google** [comment]
I wonder if the problem is that the client is timing out while the resources are spinning up. Could you look at the manager logs and see if there are erros there? Or can you look at the state of the deployment.
# Show your deployments: (depending on where you run, might need to specify --service)
dm list 

# and then look at the state of the deployment
dm get <your deployment>

My hunch is that this is related to #183

**Comment by vaikas-google** [comment]
whoah!!!, sorry about the huge font. Was not trying to shout :(

**Comment by michelleN** [comment]
```
examples [master]$ dm list
2016/01/26 10:50:16 cannot list deployments: Get http://localhost:8001/api/v1/proxy/namespaces/dm/services/manager-service:manager/deployments: net/http: request canceled (Client.Timeout exceeded while awaiting headers)
```

pod logs for manager:
```
examples [master]$ kubectl logs manager-rc-hqjh5 --namespace=dm
2016/01/26 17:17:08 Version: 0.0.1
2016/01/26 17:17:08 Listening on port 8080...
```

**Comment by michelleN** [comment]
hahaha no worries. I didn't think you were shouting, just trying to help me read better :eyes:

**Comment by jackgr** [comment]
@michelleN was this issue fixed by the fix to #183?
